### ISSUE TITLE
WMSTileLayer includes event handlers in tile URLs

### ISSUE THREAD

**Issue by markhepburn** [PaulLeCam/react-leaflet#373]
### Expected behavior

Event handlers can be specified in props to a WMSTileLayer, and attached to the leaflet element, but not included in the tile URLs.

### Actual behavior

Handlers are correctly attached, but tile URL query parameters include the on-* names, and javascript code as values.

### Steps to reproduce

Load a WMSTileLayer with a few events (eg onLoad, onTileError), for eg
https://www.webpackbin.com/bins/-KtFOnRpBwYMHZSfVgrk

Open the network inspector and observe the WMS tile requests, paying attention to the query parameters.  Depending on your browser, you should see something like (this example just using `onTileerror=console.warn`).
`onTileerror:function warn() { [native code] }`
for usage
```jsx
<WMSTileLayer
    layers='nasa:bluemarble'
    url="http://demo.opengeo.org/geoserver/ows?"
    onTileerror={console.warn}
    onLoading={console.log}
    onLoad={console.log}
    />
```

### What should happen?

I haven't attempted a patch yet, but from a skim I think [WMSTileLayer.updateLeafletElement](https://github.com/PaulLeCam/react-leaflet/blob/3aaa3376e3804a996ab89aa2510f82a4027fd619/src/WMSTileLayer.js#L26) should remove the extracted events from `params` before calling [`leafletElement.setParams(params)`](https://github.com/PaulLeCam/react-leaflet/blob/3aaa3376e3804a996ab89aa2510f82a4027fd619/src/WMSTileLayer.js#L36)

**Comment by PaulLeCam** [comment]
Should be fixed in v1.6.6, please try it out.

**Comment by markhepburn** [comment]
Looks great, thanks for your prompt response!
